src/components/ui/sidebar/organization-switcher.tsx

```diff
@@ -75,6 +79,10 @@ export function OrganizationSwitcher({
                 setOpenMobile(false);
               }
             }}
+            className={cn("gap-2 p-2", {
+              "bg-primary-500 text-white":
+                org?.name === selectedOrganization?.name,
+            })}
```

Comment:
let's follow consistent pattern for conditional classes. mixing patterns is not desired.

```suggestion
            className={cn("gap-2 p-2", org?.name === selectedOrganization?.name && "bg-primary-500 text-white")}
```